src/exts/discord_integration.js

```diff
@@ -0,0 +1,71 @@
+const { bridge } = require('../spring_api');
+const { log } = require('../spring_log');
+const DiscordRPC = require("discord-rpc");
+
+// Application ID from https://discord.com/developers/applications/<clientId>/
+const clientId = '1185990483143569448';
```

Comment:
where does it come from? If you had to register it, then this should be a config.json, and all bar specific options should be configured like that.

I would like to keep this repo without any BAR specific things, so it could be used by other Recoil games.

Reply:
This is a generated application/client ID you get when creating an application on https://discord.com/developers/applications. 

Should I add `https://maps-metadata.beyondallreason.dev/latest/discordPresenceThumb/redir.${encodeURIComponent(command.details)}.1024.jpg there as well?

Reply:
Yeah, some sort of string that can be a key or just url to template.

Reply:
Done